e2e/scaffolding/extrinsicHelpers.ts

```diff
@@ -259,7 +259,7 @@ export class Extrinsic<N = unknown, T extends ISubmittableResult = ISubmittableR
     return firstValueFrom(
       this.extrinsic()
         .paymentInfo(getUnifiedAddress(this.keys))
-        .pipe(map((info) => info.partialFee.toBigInt() * 2n))
+        .pipe(map((info) => info.partialFee.toBigInt()))
```

Comment:
This was the main hack to get a bunch of tests passing, but it was only there because I hadn't realized withdraw_fee was happening twice, and that was due to the changes to the way <del>`SignedTransactions`</del> `TransactionExtensions` work.